backend/server/mongodb/actions/Animal.ts

```diff
@@ -0,0 +1,28 @@
+import { Types } from "mongoose";
+import AnimalModel from "server/mongodb/models/Animal";
+import dbConnect from "server/utils/dbConnect";
+import { SubHandler } from "src/utils/types";
+
+export async function createAnimal(
+  totalHours: number, // TODO: user flow for totalHours (default to 0?)
```

Comment:
total hours can be optional too; we only really need the handler object id to create a service animal for them